WordPress/WordPressTest/ContextManagerMock.swift

```diff
@@ -2,7 +2,23 @@ import XCTest
 
 @testable import WordPress
 
-extension ContextManagerMock {
+class ContextManagerMock: ContextManager {
+
+    convenience init() {
+        self.init(modelName: ContextManagerModelNameCurrent)
+    }
+
+    init(modelName: String) {
+        super.init(modelName: modelName, store: URL(fileURLWithPath: "/dev/null"))
```

Comment:
Would be nice to document the use of `/dev/null` for In-Memory model a bit better here (and maybe as part of the `/// parameter storeURL` doc in the `ContextManager` superclass's `init` as well?) 🙃

Reply:
Yep, I'll add some comments to the initialiser in ContextManager.h

Reply:
A further refinement / nitpick would be to move this URL definition in a constant, to make it clearer what it does. 

Maybe something like:

```swift
super.init(modelName: modelName, store: ContextManager.inMemoryStoreURL)
```